plugin.video.nhlgcl/addon.xml

```diff
@@ -14,15 +15,17 @@
     <platform>all</platform>
     <summary lang="en">Watch out-of-market games from around the League on Kodi, with HD quality streaming</summary>
     <description lang="en">With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi</description>
-    <disclaimer lang="en"></disclaimer>
+    <disclaimer lang="en">Requires a valid NHL TV account to view full Live and Archived games. Blackout and other restrictions apply, including national blackouts.</disclaimer>
     <news>
-- Add support for inputstream adaptive
+        - Code clean up
+        - Fix error when user cancel stream select
+        - Remove listitem colors
     </news>
     <language>en</language>
     <platform>all</platform>
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
can you take the time to update this field to the spdx identifier alternative? See here: https://spdx.org/licenses/
Should be `GPL-2.0-only` or `GPL-2.0-or-later`.
Thanks